hack/preload-images/preload_images.go

```diff
@@ -33,17 +38,24 @@ const (
 )
 
 var (
-	kubernetesVersion = ""
-	tarballFilename   = ""
+	kubernetesVersion   = ""
+	tarballFilename     = ""
+	dockerStorageDriver = ""
 )
```

Comment:
Add a version for this script that generates images, each time we make a fundemental change to this script, till be bumped by one.
also we should be able to pass a commit sha so in future we can build the Tar for the PR in github actions and test the same Tar in github actions for that speceific commit. (6 digit commit)

Reply:
cool, i can open an issue for that to track it